server/utils/nunjucksSetup.ts

```diff
@@ -48,6 +49,10 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
     return `${array[0][0]}. ${array.reverse()[0]}`
   })
 
+  njkEnv.addFilter('formatName', (name: string) => {
+    return convertToTitleCase(name)
+  })
```

Comment:
we tend to handle this on the backend for arrivals - we should probably do the same for temp absences

Reply:
Removed on latest commit